stable/prometheus-cloudwatch-exporter/values.yaml

```diff
@@ -106,3 +107,9 @@ readinessProbe:
   timeoutSeconds: 5
   successThreshold: 1
   failureThreshold: 3
+
+serviceMonitor:
+  enable: false
+  scrapeInterval: 60s
+  scrapeTimeout: 59s
```

Comment:
It is a better idea to inherit the scrapeInterval and scrapeTimeout from the host prometheus than set them to specific values that may be different from other service monitors and make reasoning about intervals more difficult

Reply:
add condition to define this variables only if needed